source/native/symbulation.cc

```diff
@@ -31,7 +31,7 @@ EMP_BUILD_CONFIG(SymConfigBase,
     VALUE(FILE_NAME, string, "_data_", "Root output file name")
 )
 
-int symbulation_main(int argc, char * argv[])
+int symbulation_main(int argc, char * argv[]) // personalized main for testing 
```

Comment:
This comment is confusing since this isn't the main for testing, let's just remove it.